src/model/beneficial.owner.type.model.ts

```diff
@@ -1,12 +0,0 @@
-export const BeneficialOwnerTypeKey = "beneficial_owner_type";
-
-export enum BeneficialOwnerTypeChoice {
-    individual = "individualOwner",
-    otherLegal = "otherLegalOwner",
```

Comment:
`src/model/beneficial.owner.type.model.ts` is unrelated to the current trust work. I don't think the file should be removed/split at this point as it's out of scope for the current task